<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54)</string>
  </property>
  <widget class="QComboBox" name="brand">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 168, 167)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="model">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 168, 167)</string>
   </property>
  </widget>
  <widget class="QLabel" name="brand_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Бренд:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Модель:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_but">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 168, 167); font-size:11pt;</string>
   </property>
   <property name="text">
    <string>Показать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number_user">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(243,243,243)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Номер пользователя:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Кол-во суток:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cout_hour">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(243,243,243)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Номер автомобиля:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number_auto">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(243,243,243)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>К оплате</string>
   </property>
  </widget>
  <widget class="QLabel" name="for_payment">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="creat_order_but">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 168, 167); font-size:11pt;</string>
   </property>
   <property name="text">
    <string>Добавить заказ</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>731</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Номер</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Цена</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Локация</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>340</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(167, 168, 167); font-size:11pt;</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
